frontend/src/pages/connectionTypes/columns.ts

```diff
@@ -26,18 +26,15 @@ const sorter = (
   }
 
   if (keyField === 'enable') {
-    return a.metadata.annotations?.['opendatahub.io/enabled'] === 'true' ||
-      b.metadata.annotations?.['opendatahub.io/enabled'] !== 'true'
-      ? -1
-      : 1;
+    return a.metadata.annotations?.['opendatahub.io/disabled'] !== 'true' ? -1 : 1;
```

Comment:
Not a change from before but do we want to sort by name if the enabled state is the same?

Reply:
ya that's probably a good idea!